AdyenNetworking/APIClient/RetryOnErrorAPIClient.swift

```diff
@@ -36,19 +36,4 @@ public final class RetryOnErrorAPIClient: APIClientProtocol {
             }
         }, completionHandler: completionHandler)
     }
-    
-    /// :nodoc:
-    public func perform<R>(
-        _ request: R,
-        completionHandler: @escaping CompletionHandler<R.ResponseType>
-    ) where R : Request, R.ResponseType == DownloadResponse {
-        apiClient.perform(request, shouldRetry: { result in
-            switch result {
-            case .success:
-                return false
-            case .failure:
-                return true
-            }
-        }, completionHandler: completionHandler)
-    }
```

Comment:
I am sorry, don't understand why remove it?

Reply:
To avoid the duplication when it's not needed at this moment. I think in this case my eagerness to make it thorough is unnecessary.

Reply:
In other words for now the simple download api wrapper is sufficient. If in future the retry wrapper is needed, it can then be added.